api/locale/ko.yml

```diff
@@ -0,0 +1,3 @@
+ko:
+  view: "1 조회수"
+  views: "%{number} 조회수"
```

Comment:
From YouTube it looks like the format they use is like this:

![Screenshot_20231105-115215.png](https://github.com/DenverCoder1/github-readme-youtube-cards/assets/20955511/98ddae4d-843f-42c9-b433-52cd1a5cecf5)

Does this look right to you?

```suggestion
  view: "조회수 1회"
  views: "조회수 %{number}회"
```

Reply:
I agree with your suggestion, but I am not sure about the placeholder for the appropriate unit. 
For eaxmple:
`조회수 {view_count} {unit}` 
the unit placeholder will change accordingly to the view count. 

YouTube in Korean primarily uses the following units for representing view counts:

- "만회" for Thousands: Example - "7.3천회" means "7,300 views."
- "천만회" for Tens of Millions: Example - "8826만회" means "8,826,000 views."
- "회" for Regular Views: Example - "799회" means "799 views."

These are the common units used on YouTube in the Korean language to make view counts more easily understandable to Korean viewers.

So, I suggest the follow:

```yml
ko:
  view: "조회수 1"
  views: "조회수 %{number}"
```

Reply:
I have read through the instructions provided in the `CONTRIBUTING.md`, and I have noticed that you are using the command `python` instead of `python3`, even though in `action.yml`, you mention `python-version: "3.10"`. So my question is, do you have a symbolic link set up for the `python` command to point to `Python 3.10` on your system?

Reply:
I am trying to check out the result locally, but it won't work. Even when I run `tox` I get 10 failed tests mainly because of `ssl certificate verify failed`

I was thinking about opening an issue ticket about it, but I was not sure if this is caused because of my local setup. Maybe, the setup instructions might need some modification. 

<img width="246" alt="image" src="https://github.com/DenverCoder1/github-readme-youtube-cards/assets/39487200/9556faed-5c34-468f-9c93-9e77d7146f24">

Reply:
This may help with that issue - https://stackoverflow.com/questions/27835619/urllib-and-ssl-certificate-verify-failed-error

I have it installed on Arch Linux so I believe both `python` and `python3` work for me, but on each system the command may be different.

There's also a preview deployment here:

https://ytcards-git-fork-eliasafara-yt-cards-korean-378315-denvercoder1.vercel.app/?id=6u9BrDaSHJc&title=Automatically+Deploy+to+Fly.io+with+GitHub+Actions&lang=ko&timestamp=1661889604&background_color=%230d1117&title_color=%23ffffff&stats_color=%23dedede&width=250&border_radius=5&duration=312&max_title_lines=1

![](https://ytcards-git-fork-eliasafara-yt-cards-korean-378315-denvercoder1.vercel.app/?id=6u9BrDaSHJc&title=Automatically+Deploy+to+Fly.io+with+GitHub+Actions&lang=ko&timestamp=1661889604&background_color=%230d1117&title_color=%23ffffff&stats_color=%23dedede&width=250&border_radius=5&duration=312&max_title_lines=1)

Reply:
> YouTube in Korean primarily uses the following units for representing view counts

Is there any case where the unit is not followed by `회`?

Reply:
Thanks for the link, I will check it out. As for your question, I don't think there is.